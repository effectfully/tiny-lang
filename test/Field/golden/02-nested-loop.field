let result = 0;
for i = 1 to 2 do
    for j = 1 to 3 do
        let result = result + i * j;
    end;
end;
assert result == 18;
