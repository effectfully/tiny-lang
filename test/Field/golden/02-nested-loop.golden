let result_3 = 0;
let i_4 = 1;
let j_5 = 1;
let result_6 = result_3 + (i_4 * j_5);
let j_7 = 2;
let result_8 = result_6 + (i_4 * j_7);
let j_9 = 3;
let result_10 = result_8 + (i_4 * j_9);
let i_11 = 2;
let j_12 = 1;
let result_13 = result_10 + (i_11 * j_12);
let j_14 = 2;
let result_15 = result_13 + (i_11 * j_14);
let j_16 = 3;
let result_17 = result_15 + (i_11 * j_16);
assert result_17 == 18;
